src/hiero_sdk_python/transaction/transfer_transaction.py

```
@@ -91,11 +96,12 @@ def _add_hbar_transfer(
                 transfer.amount += amount
                 return self
 
-        self.hbar_transfers.append(
-            HbarTransfer(account_id, amount, is_approved))
+        self.hbar_transfers.append(HbarTransfer(account_id, amount, is_approved))
         return self
 
-    def add_hbar_transfer(self, account_id: AccountId, amount: Union[int, Hbar]) -> "TransferTransaction":
+    def add_hbar_transfer(
+        self, account_id: AccountId, amount: Union[int, Hbar]
```

Ruff (0.14.14)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)